compiler-rt/include/profile/InstrProfData.inc

```diff
@@ -781,6 +798,9 @@ serializeValueProfDataFrom(ValueProfRecordClosure *Closure,
 #define INSTR_PROF_COVNAME_COFF ".lcovn"
 #define INSTR_PROF_ORDERFILE_COFF ".lorderfile$M"
 
+// TODO: Placeholder for Windows. We need to revise when we upstream this.
```

Comment:
Comment requires update (this PR is for "upstream"). Perhaps the TODO should be made a FIXME.

Reply:
Thanks! Updated!